course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -215,91 +401,165 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "1\\. Load as cubes the datasets found at `iris.sample_data_path('E1_north_america.nc')` and `iris.sample_data_path('A1B_north_america.nc')`. Print the summary of each cube."
+    "#### 1\\. Load data\n",
+    "Load as cubes the datasets found at `iris.sample_data_path('E1_north_america.nc')` and `iris.sample_data_path('A1B_north_america.nc')`. Print the summary of each cube."
```

Comment:
The questions says to ` Print the summary of each cube.` but then the sample solution does a full print out of the one cube and a summary of the other:
```
cube_e1 = iris.load_cube(iris.sample_data_path('E1_north_america.nc'))
cube_a1b = iris.load_cube(iris.sample_data_path('A1B_north_america.nc'))
print(cube_e1)
print('------')
print(cube_a1b.summary(shorten=True))
```